letsencrypt/README.rst

```diff
@@ -97,6 +114,13 @@ an upstream for your odoo instance and do something like::
         proxy_pass    http://yourodooupstream;
     }
 
+If you're using a multi-database installation (with or without dbfilter option)
```

Comment:
But we usually have several DBs under the same dbfilter (real DB and test DB for example), and with this, it works also for this case. And you don't need here the `env` for anything more than to call some methods, so please accept the change.